exporters/common/src/main/java/io/opentelemetry/exporter/grpc/GrpcSenderConfig.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.grpc;
+
+import io.opentelemetry.exporter.compressor.Compressor;
+import io.opentelemetry.sdk.common.export.RetryPolicy;
+import java.io.OutputStream;
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.X509TrustManager;
+
+/**
+ * Configuration for {@link GrpcSender} implementations, provided via {@link
+ * GrpcSenderProvider#createSender(GrpcSenderConfig)}.
+ */
+@Immutable
+public interface GrpcSenderConfig {
+
+  /**
+   * The gRPC endpoint to send to, including scheme. Omits path, which must be constructed from
+   * {@link #getFullServiceName()} and {@link #getMethodName()}.
+   */
+  URI getEndpoint();
+
+  /**
+   * The fully qualified gRPC service name, e.g. {@code
+   * opentelemetry.proto.collector.trace.v1.TraceService}.
+   */
+  String getFullServiceName();
+
+  /** The gRPC method name, e.g. {@code Export}. */
+  String getMethodName();
```

Comment:
could combine these two properties (we are doing that in https://github.com/open-telemetry/semantic-conventions/pull/3223), and I think(?) I saw that the underlying grpc API does the same